tests/test_functions/events/main.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google LLC
```

Comment:
This file should probably be moved to `tests/test_functions/cloudevents/main.py`.